index.php

```diff
@@ -124,8 +124,17 @@
 		}
 		elseif(my_strpos($user['sid'], 'bot=') !== false && $spiders[$botkey])
 		{
+			if($mybb->settings['wolorder'] == 'username')
+			{
+				$key = $spiders[$botkey]['name'];
+			}
+			else
+			{
+				$key = $user['time'];
+			}
+
```

Comment:
Why you need to sort here by key? The query result is already sorted from database. If its failing to sort there the issue should be fixed in query itself...